src/test/java/api/queue/RequestQueueResourceTest.java

```diff
@@ -223,6 +231,101 @@ void refuseAttemptToMoveRequestBeingFulfilledFromOneOfTheTopPositionsWhenTlrEnab
       is("Requests can not be displaced from top positions when fulfillment begun."));
   }
 
+  @Test
+  void shouldGetRequestQueueForItemSuccessfully() {
+    UUID facultyGroupId = patronGroupsFixture.faculty().getId();
+    UUID staffGroupId = patronGroupsFixture.staff().getId();
+    UUID isbnIdentifierId = identifierTypesFixture.isbn().getId();
+    String isbnValue = "9780866989732";
+    item = itemsFixture
+      .basedUponSmallAngryPlanet(
+      identity(),
+      instanceBuilder -> instanceBuilder.addIdentifier(isbnIdentifierId, isbnValue).withId(instanceId),
+      itemBuilder -> itemBuilder
+        .withCallNumber("itCn", "itCnPrefix", "itCnSuffix")
+        .withEnumeration("enumeration1")
+        .withChronology("chronology")
+        .withVolume("vol.1")
+        .withCopyNumber("1"));
+
+    final IndividualResource sponsor = usersFixture.rebecca(user -> user
+      .withPatronGroupId(facultyGroupId));
+
+    final IndividualResource proxy = usersFixture.steve(user -> user
+      .withPatronGroupId(staffGroupId));
+
+    final IndividualResource cd1 = servicePointsFixture.cd1();
+
+    UUID pickupServicePointId = cd1.getId();
```

Comment:
If we're just using this var once, can it be inlined?

Reply:
fixed